incubator/hnc/pkg/controllers/config_controller.go

```diff
@@ -42,13 +56,24 @@ func (r *ConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	// achieve the reconciliation.
 	r.Log.Info("Reconciling cluster-wide HNC configuration.")
 
+	// TODO: This is only for debuging and testing in the early development stage. Remove this when the
+	// implementation is completed.
+	r.logSpec(inst)
+
+	// Create cooresponding ObjectReconcilers for newly added types, if needed.
+	// TODO: Returning an error here will make controller-runtime to requeue the object to be reconciled again.
```

Comment:
I think we shouldn't return the error in this PR - just log it, and return nil. Just because you have an error doesn't mean you _have_ to return it (though lots of golang coders behave that way!)

Reply:
Done.